.dockerignore

```diff
@@ -0,0 +1,8 @@
+.circleci/
```

Comment:
I think you can remove everything except a line for pycache now that you're only copying what you need into the working dir

Reply:
I could, but docker will still send everything in the working directory to the build context which will add more time to the build. There's more about the build context [here](https://docs.docker.com/v17.09/engine/reference/builder/#usage).

Reply:
Ahh interesting ok 👍